response_operations_ui/assets/scss/includes/_overrides.scss

```diff
@@ -0,0 +1,343 @@
+html{
```

Comment:
Basically no need to review this - it's copied across to a new location, but is old code.  I will be refactoring it later.